commercetools-models/src/main/java/io/sphere/sdk/customergroups/CustomerGroup.java

```diff
@@ -47,6 +51,10 @@
     String getName();
 
     @Override
+    @Nullable
```

Comment:
Here we could already use our existing update action generator or am I wrong? Cristian added a `@HasUpdateAction` annotation and we should try to use it here.